src/extension_instance.mli

```diff
@@ -2,7 +2,7 @@ open Import
 
 type t
 
-val make : unit -> t
+val make : ?codelens:bool -> ?extended_hover:bool -> unit -> t
```

Comment:
Please don't use optional arguments unless there's a specific reason

Reply:
> LanguageClient.OcamllspSettings.create

That one is weird as well. It's our own private API, what's it doing inside the vscode API bindings?

> But now that I think about it, this is a binding and that's likely the reason why it is using optional parameters.

What do you mean by "binding"?